ansible/vars/flatcar/flatcar.yaml

```diff
@@ -8,8 +8,6 @@ sysusr_prefix: /opt
 sysusrlocal_prefix: /opt
 systemd_prefix: /etc/systemd
 
-containerd_cri_socket: /run/docker/libcontainerd/docker-containerd.sock
-
```

Comment:
Why is this being removed? Is it being defined somewhere else?

Reply:
Yes, the default comes from https://github.com/mesosphere/konvoy-image-builder/blob/main/ansible/group_vars/all/defaults.yaml#L22

Reply:
And https://github.com/mesosphere/konvoy-image-builder/pull/66/files#diff-f1a7c1d253313ff70bf0da17787af9de98657330662cbb515b6fe38c1ded0bbcR15 then makes sure whoever still expects a docker socket, will find it.